src/coreclr/src/inc/clrprovidersdummy.h

```diff
@@ -0,0 +1,534 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+typedef struct _EVENT_DESCRIPTOR
```

Comment:
It does not look right to need everything in this file with `FEATURE_EVENT_TRACE` disabled? What it would take to ifdefs uses instead?

Reply:
It only took few `#ifdef`s in `eventtracebase.h` to remove this file.